### PR TITLE
Increased by 2 the buffer size for the WebAudio spec

### PR DESCRIPTION
**Pull request by plehegar**

### CODE REVIEW

**Comment by tripu** [comment]
This change LGTM. Although I'm not familiar enough with Spec Generator and ReSpec to be sure there won't be problems associated with doubling the buffer size, so I'll let somebody else review it.

**Comment by plehegar** [comment]
@deniak any memory limit we could run into on the server?

File: generators/respec.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ exports.generate = function (url, params, cb) {
     execFile(
         phantom.path
     ,   ["--ssl-protocol=any", r2hPath, url]
-    ,   { maxBuffer: 800*1024 } // default * 2
+    ,   { maxBuffer: 1600*1024 } // default * 2
```
Comment (deniak) [inline generators/respec.js]: The inline comment says `default * 2`. It should be `default * 8`

**Comment by deniak** [comment]
The size is still reasonable so it's ok to double the `maxBuffer`. However, as specs keep growing and growing, it would be better to switch from `execFile` to `spawn`. I can try to take care of it if time allows (https://github.com/w3c/spec-generator/issues/22).
Please address the little comment I made before merging the MR.

**Pull request merged**